terraform.gpaas-azure-migration/key-vault-tfvars-secrets.tf

```diff
@@ -0,0 +1,54 @@
+resource "azurerm_key_vault" "tfvars" {
```

Comment:
Just for my understanding how do the secrets get added to this?

Reply:
This particular file is responsible for setting up an Azure Key Vault that will contain the `.tfvars` file for each environment.

The actual `.tfvars` file contains a key:value variable `container_secret_environment_variables` of which you can store all of the secrets that you wish to be passed into the running container. All of the values included in this variable are considered sensitive secrets in Azure.

In order to update _those_ secrets, you'd need to first pull down the existing `tfvars` from the Azure Key Vault, then make your change, then run `terraform apply --var-file=XXX.tfvars` to deploy the change to the container app, as well as syncing your local changes up to the Key Vault.

This is something that we can document for you further down the line.